Runtimes/Unity/BepInEx.Unity.Common/ConfigUnityLog.cs

```diff
@@ -0,0 +1,16 @@
+﻿using BepInEx.Configuration;
+
+namespace BepInEx.Unity.Common;
+
+internal static class ConfigUnityLog
+{
+    internal static readonly ConfigEntry<bool> ConfigUnityLogging = ConfigFile.CoreConfig.Bind(
+     "Logging", "UnityLogListening",
+     true,
+     "Enables showing unity log messages in the BepInEx logging system.");
+
+    internal static readonly ConfigEntry<bool> ConfigDiskWriteUnityLog = ConfigFile.CoreConfig.Bind(
+     "Logging.Disk", "WriteUnityLog",
+     false,
```

Comment:
I think this should be on by default, many newer games don't even output PlayerLog and it's annoying to put the two logs together time-wise afterwards.

Reply:
I think the question is more why was it off by default in the first place?

I was thinking it had to do with games spamming this and if that was why...I don't see how that changes? The game would still have that problem if so with v7

Reply:
After discussion, I am fine with this so I changed it to true by default.